src/pedal/config.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{pedal, Pedal as PedalTrait, Result};
+use cpal::StreamConfig;
+use serde::Deserialize;
+use Pedal::*;
```

Comment:
The imports the cases of the `Pedal` enum declared further down in this file so that you can use them without prefixing `Pedal::`. eg. `Pedal::Transparent` becomes `Transparent`.

Reply:
I think the term "pedal" is getting overloaded in a way that's making it harder to follow what's going on - it seems like there are "pedal configs" (here) vs "primitives"/"units" (or maybe some less generic name)?

The fact that you had to rename the import above to `PedalTrait` to avoid namespace clashing seems like evidence of that